fleet/models.py

```diff
@@ -22,17 +21,15 @@ def format_phone_number(self):
         return
 
     def get_name(self):
-        name = 'Station ' + self.station_id
-        return name
+        return f'Station {self.pk}'
 
     def get_logo(self):
         if self.logo:
             return self.logo.url
         return static('fleet/logo_jcc.png')
 
     def get_path(self):
```

Comment:
Where/how do you use this?

Reply:
https://github.com/lemmetry/jcc/blob/ca6faf11d0c512d4fa52189261abb32bd72ac4a8/inventory/views.py#L27
https://github.com/lemmetry/jcc/blob/ca6faf11d0c512d4fa52189261abb32bd72ac4a8/inventory/views.py#L50
https://github.com/lemmetry/jcc/blob/ca6faf11d0c512d4fa52189261abb32bd72ac4a8/inventory/views.py#L78
https://github.com/lemmetry/jcc/blob/ca6faf11d0c512d4fa52189261abb32bd72ac4a8/inventory/views.py#L184
https://github.com/lemmetry/jcc/blob/ca6faf11d0c512d4fa52189261abb32bd72ac4a8/inventory/views.py#L241